### DISCUSSION TITLE
Background remove like in https://www.remove.bg/

### DISCUSSION THREAD

**Discussion by vkkindia** [libvips/libvips#3541] [Help wanted]
I came across this website https://www.remove.bg/ which removed background using their algorithm. can we do this in libvips. either in python or C

**Comment by jcupitt** [comment]
Their BG removal is AI, so it's not something that libvips can do directly.

**Comment by vkkindia** [comment]
Yes, Vips does wonders may be in future versions AI might be added.
In python rembg  (https://github.com/danielgatis/rembg) almost does the
same.
